src/send/ValidateRecipientAccount.tsx

```diff
@@ -272,7 +273,7 @@ export class ValidateRecipientAccount extends React.Component<Props, State> {
         : singleDigitInputValueArr.filter((entry) => /[a-f0-9]/gi.test(entry)).length === 4
 
     return (
-      <View style={styles.container}>
+      <SafeAreaView style={styles.container}>
```

Comment:
this change was unrelated (noticed it while testing the flow) but small so i kept it